settings.gradle.kts

```diff
@@ -66,3 +67,7 @@ project(":spark-connector:spark-runtime-3.5").projectDir = file("spark-connector
 include("web")
 include("docs")
 include("integration-test-common")
+include("flink-connector:flink")
+findProject(":flink-connector:flink")?.name = "flink"
+include("flink-connector:flink-runtime")
+findProject(":flink-connector:flink-runtime")?.name = "flink-runtime"
```

Comment:
what's the purpose of setting the name?  I failed to run `./gradlew flink-runtime:build` is it expected?